### PR TITLE
Move to redis broker

### PR DESCRIPTION
**Pull request by AlviseSembenico**

### CODE REVIEW

File: src/vng/conf/base.py [hunk 1]
```diff
@@ -317,6 +317,7 @@
 ENVIRONMENT = None
 SHOW_ALERT = True
 
+CELERY_BROKER_URL = 'redis://localhost'
```
Comment (alextreme) [inline src/vng/conf/base.py]: I suspect you need to add the Redis Database number, for instance based on SLA dashboard: 

CELERY_BROKER_URL = "redis://127.0.0.1:6379/12"

It's best to use a different Redis database (so specify this in the test.py and staging.py settings with a different number)

Comment (AlviseSembenico) [reply src/vng/conf/base.py]: Updated, is this way correct?

File: src/vng/conf/test.py [hunk 2]
```diff
@@ -73,6 +73,7 @@
 # Custom settings
 #
 
+CELERY_BROKER_URL = "redis://127.0.0.1:6379/12"
```
Comment (alextreme) [inline src/vng/conf/test.py]: This will clash with the Redis task-queue for SLA dashboard. I'd use 13 + 14 if I were you.

Yes this is annoying, sorry :)

Comment (AlviseSembenico) [reply src/vng/conf/test.py]: Updated

File: src/vng/conf/base.py (hunk 1 above)
Comment (alextreme) [reply src/vng/conf/base.py]: Yes, thanks! Give it a try on vng-test

**Pull request merged**